src/main/drivers/serial_uart.h

```diff
@@ -25,6 +25,24 @@
 // Size must be a power of two due to various optimizations which use 'and' instead of 'mod'
 // Various serial routines return the buffer occupied size as uint8_t which would need to be extended in order to
 // increase size further.
+#ifdef USE_MSP_DISPLAYPORT
+#define UART1_RX_BUFFER_SIZE    256
+#define UART1_TX_BUFFER_SIZE    1280
```

Comment:
@ligenxxxx why 1280 and not 1024? The comment above mentions that buffer should the the power of 2

Reply:
If there are so many osd elements selected, 1024 depth is not enough to use, but 2048 is too much, how to choose?